app/models/invenio_rdm_record_converter.rb

```diff
@@ -239,6 +239,14 @@ def build_creator_contributor_json(creator)
       }
     # Personal record with user in database OR is a person's formal name
     elsif dh_user.present? || creator.include?(",")
+      creatibutor_json = {
+        "person_or_org": {
+          "given_name": "given_name",
+          "family_name": "family_name",
+          "type": "personal"
+        }
+      }
```

Comment:
Is this supposed to be in here?

Reply:
omg no :(

Reply:
ahhhh thanks @sharpattack  - this would have been disastrous

Reply:
I see this was merged. Did this change end up being needed?